var/spack/repos/builtin/packages/py-pymol/package.py

```diff
@@ -15,29 +15,42 @@ class PyPymol(PythonPackage):
     homepage = "https://pymol.org"
     url      = "https://github.com/schrodinger/pymol-open-source/archive/v2.4.0.tar.gz"
 
+    version('2.5.0', sha256='aa828bf5719bd9a14510118a93182a6e0cadc03a574ba1e327e1e9780a0e80b3')
     version('2.4.0', sha256='5ede4ce2e8f53713c5ee64f5905b2d29bf01e4391da7e536ce8909d6b9116581')
     version('2.3.0', sha256='62aa21fafd1db805c876f89466e47513809f8198395e1f00a5f5cc40d6f40ed0')
 
-    depends_on('python+tkinter', type=('build', 'run'))
-    depends_on('freetype', type=('build', 'run'))
-    depends_on('glew', type=('build'))
-    depends_on('glm', type=('build'))
-    depends_on('py-numpy', type=('build', 'run'))
-    depends_on('tcsh', type=('build', 'run'))
+    depends_on('python+tkinter@2.7:', type=('build', 'run'), when='@2.3.0:2.4.0')
+    depends_on('python@3.6:', type=('build', 'run'), when='@2.5.0:')
```

Comment:
Is tkinter no longer required? I still see a lot of imports

Reply:
my understanding is that they've moved away from Tk since 2.2.0 - seems like the plugins haven't all kept up though. I'll add tkinter back.